horton/grid/cext.pyx

```diff
@@ -1186,11 +1186,31 @@ cdef class UniformGrid(object):
         assert shape.shape[0] == 3
         assert pbc.flags['C_CONTIGUOUS']
         assert pbc.shape[0] == 3
-
         self._this = <uniform.UniformGrid*>(new uniform.UniformGrid(
             &origin[0], &grid_rvecs[0, 0], &shape[0], &pbc[0],
         ))
 
+    def __init__(self, np.ndarray[double, ndim=1] origin not None,
+                 np.ndarray[double, ndim=2] grid_rvecs not None,
+                 np.ndarray[long, ndim=1] shape not None,
+                 np.ndarray[long, ndim=1] pbc not None):
+        """__init__(self, ndarray origin, ndarray grid_rvecs, ndarray shape, ndarray pbc)
```

Comment:
See issue #101 for the details. In short: cython normally adds such signatures automatically with the `embedsignature` option but it fails to do so for constructors.